comp/core/config/config.go

```diff
@@ -69,10 +78,18 @@ func NewServerlessConfig(path string) (Component, error) {
 	return newConfig(d)
 }
 
-func newConfig(deps dependencies) (Component, error) {
+func newConfigProvide(deps dependencies) (provides, error) {
+	c, err := newConfig(deps)
+	return provides{
+		Comp:          c,
+		FlareProvider: flaretypes.NewProvider(c.fillFlare),
+	}, err
+}
+
+func newConfig(deps dependencies) (*cfg, error) {
```

Comment:
Is this method use anywhere else or can we merge it with `newConfigProvide` ?

If it's use somewhere else, then we need to double check the flare provider as it won't be loaded there.

Reply:
The only place where it is not combined with the FlareProvider is for Serverless, which don't have flare component.